deploy/arbitrum/230_permissionless_deployments.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from "dotenv"
+import { DeployFunction } from "hardhat-deploy/types"
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { deployPermissionlessPoolComponents } from "../deployUtils"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  dotenv.config()
+  const { deployments, getNamedAccounts, getChainId, ethers } = hre
+  const { deploy, get, getOrNull, execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  deployPermissionlessPoolComponents(
+    hre,
+    (await get("MetaSwapDeposit")).address,
+  )
```

Comment:
Since this is an async function, await should be used in order for deploy scripts to run in sequence, not parallel.